CipherText/Rotator/RotatorBuilder.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjose00
+ * Date: 11/14/17
+ * Time: 3:54 PM
+ */
+
+namespace Giftcards\Encryption\CipherText\Rotator;
+
+use Giftcards\Encryption\CipherText\Rotator\Store\StoreRegistry;
+use Giftcards\Encryption\CipherText\Rotator\Store\StoreRegistryBuilder;
+use Giftcards\Encryption\Encryptor;
+use Giftcards\Encryption\EncryptorBuilder;
+use Giftcards\Encryption\Factory\BuilderInterface;
+
+class RotatorBuilder
+{
+
+    // <editor-fold desc="Construction and Building">
```

Comment:
adding a `newInstance` method here will make this consistent with the other builders